test/data/html/tests.js

```diff
@@ -900,6 +900,52 @@ exports.test_data = {
             unchanged: '<html>\n\n<head>\n<meta>\n</head>\n\n</html>'
         }]
     }, {
+        name: "content_unformatted to prevent formatting content",
+        description: "",
+        options: [
+            { name: 'content_unformatted', value: "['script', 'style', 'p', 'span', 'br']" }
```

Comment:
Add some examples with `<pre>` tag (as mentioned above).
Then please do the same inputs with the default  `content_unformatted`.  It is important to understand how this changes the outputs.